swagger-templates/php/composer.mustache

```diff
@@ -16,11 +16,18 @@
             "homepage": "https://squareup.com/developers"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/phpish/link_header.git"
```

Comment:
Are we sure we want to add this as a dependency? In my mind:

- We add a 4 year stale repo that could be deleted at any moment to our dependencies. 
- We add additional complexity for our end users having to download another repo.
-  Almost half of our PHP users do not use composer, how are they going get this repo's code? 

Looking at that repo, it is ~100 lines of code. I would feel more comfortable committing that directly, or rewriting that in the batch token parsing function.